.github/workflows/flox-channels.yml

```diff
@@ -0,0 +1,101 @@
+name: "Flox channel progression"
+
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Run script every day at 8 UTC
+    - cron:  '* 8 * * *'
+
+jobs:
+  flox-channels:
+    env:
+      GH_TOKEN: ${{ secrets.NIX_GIT_TOKEN_TEMP }}
+    if: github.repository_owner == 'flox'
+    runs-on: ubuntu-latest
+
+    # TODO: implement configurable merge based on matrix
+
+    name: "Periodic pull from upstream"
+    steps:
+      - name: Configure Git Env
+        run: |
+          git config --global user.email "automations@floxdev.com"
```

Comment:
We have a "technology@floxdev.com" group alias but not an "automations" one ... are you wanting to create that or should we just use technology?